src/cascadia/TerminalSettingsModel/KeyChordSerialization.h

```diff
@@ -17,6 +18,15 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     };
 }
 
+template<>
+struct Microsoft::Terminal::Settings::Model::JsonUtils::ConversionTrait<winrt::Microsoft::Terminal::Control::KeyChord>
+{
+    winrt::Microsoft::Terminal::Control::KeyChord FromJson(const Json::Value& json);
+    bool CanConvert(const Json::Value& json);
+    Json::Value ToJson(const winrt::Microsoft::Terminal::Control::KeyChord& val);
+    std::string TypeDescription() const;
+};
+

```

Comment:
@DHowett Here's the beginning of a transition to adding more `ConversionTrait`s. Hope this looks ok to you.